src/comp/pages/CreateProjectPage.tsx

```diff
@@ -26,36 +26,64 @@ class CreateProjectPage extends React.Component<
     };
   }
 
+  checkBlankSpace = (value: string) => {
+    let isError = true;
+    if (value.match(/\S/g)) {
+      isError = false;
+    }
+    return isError;
+  };
+
   handleNameInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     // multiline属性を消すと表示が乱れるのでこっちで複数行入力を阻止する
     if (e.target.value.endsWith("\n")) return;
+    const isError = this.checkBlankSpace(e.target.value);
     this.setState({
       name: e.target.value,
-      isNameError: e.target.value.length === 0,
+      isNameError: isError,
     });
   };
 
   handleIntroductionInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const isError = this.checkBlankSpace(e.target.value);
     this.setState({
       introduction: e.target.value,
-      isIntroductionError: e.target.value.length === 0,
+      isIntroductionError: isError,
     });
   };
 
   handleSendButton = () => {
-    const isNameError = this.state.name.length === 0;
-    const isIntroductionError = this.state.introduction.length === 0;
-    this.setState({
-      isNameError: isNameError,
-      isIntroductionError: isIntroductionError,
-    });
-    if (isNameError || isIntroductionError) return;
-
     // TODO: ここでAPIに情報をぶん投げる
     const projectId = "7438921";
     this.props.history.push(`/projects/detail/${projectId}`);
   };
 
+  changeButtonType = () => {
+    const ableSendFlag =
+      !this.checkBlankSpace(this.state.name) &&
+      !this.checkBlankSpace(this.state.introduction);
+    const ableSend = (
+      <Button
+        variant="contained"
+        color="secondary"
+        size="large"
+        onClick={this.handleSendButton}
+      >
+        作成
+      </Button>
+    );
+    const disableSend = (
+      <Button variant="contained" size="large" disabled>
+        作成
+      </Button>
+    );
+    if (ableSendFlag) {
+      return ableSend;
+    } else {
+      return disableSend;
+    }
```

Comment:
冗長な `else` なので消してください
(人によっては `? :` を使えと言うかもしれませんが、私は別に `if` でいいです)
```suggestion
    return disableSend;
```